patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1911,10 +1911,12 @@
  			dashType = 5;
  		}
  
-@@ -12581,11 +_,16 @@
+@@ -12580,12 +_,17 @@
+ 			setBonus = Language.GetTextValue("ArmorSetBonus.ObsidianOutlaw");
  			minionDamage += 0.15f;
  			whipRangeMultiplier += 0.3f;
- 			float num17 = 1.15f;
+-			float num17 = 1.15f;
++			float num17 = 0.15f;
```

Comment:
Definitely do the `num17 - 1` thing you suggested. 
Note the commented out vanilla code below. 
To change a number right next to commented out vanilla code would give the illusion that the value was not changed by tML and be confusing.